### PR TITLE
Feat/request tagger

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

BREAKING CHANGE: `@ovh-ux/manager-core` has now a dependency to
`@ovh-ux/ng-ovh-request-tagger`

e2c9a5a3a — feat: add request-tagger module
108adf6f2 — feat(core): add request-tagger interceptor
164a999f2 — docs: update modules list

### CODE REVIEW

**Comment by cbourgois** [comment]
I will change commit `build: add ng-ovh-request-tagger dependency` to another thing (`feat`, to create a release of `@ovh-ux/manager-core`)

File: packages/manager/modules/request-tagger/src/factory.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        taggedConfig = {
+          ...config,
+          headers: omit(config.headers, [
+            HEADER_NAVIGATION_ID,
+            HEADER_REQUEST_ID,
+            HEADER_PAGE,
+          ]),
+        };
+      }
+      return taggedConfig;
```
Comment (marie-j) [inline packages/manager/modules/request-tagger/src/factory.js]: You could `return` directly and avoid the `else` 
```suggestion
      if (find(ROUTES_PREFIX, route => startsWith(config.url, route))) {
        requestIndex += 1;
        const overridenHeaders = find(
          ROUTES_HEADERS_OVERRIDE,
          (value, route) => new RegExp(route).test(config.url),
        ) || {};

         return {
          ...config,
          headers: {
            ...config.headers,
            [HEADER_REQUEST_ID]: `${Date.now()}-${requestIndex}`,
            ...overridenHeaders,
          },
        }
        
        return {
          ...config,
          headers: omit(config.headers, [
            HEADER_NAVIGATION_ID,
            HEADER_REQUEST_ID,
            HEADER_PAGE,
          ]),
        };
      
```

**Comment by ovh-cds** [comment]
CDS Report pci#4457.0 ■   
* 
  * Build CA ✘  
  * Build EU ■   
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report telecom-dashboard#4457.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report public-cloud#4457.0 ■   
* 
  * Build CA ■   
  * Build EU ✔ 
  * Build US ✔

**Comment by ovh-cds** [comment]
CDS Report vrack#4457.0 ■   
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ■

**Comment by ovh-cds** [comment]
CDS Report overthebox#4457.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report order-tracking#4457.0 ■   
* 
  * Build CA ■   
  * Build EU ✔ 
  * Build US ✔

**Comment by ovh-cds** [comment]
CDS Report freefax#4457.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

File: packages/manager/modules/request-tagger/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "directory": "packages/manager/modules/ng-ovh-request-tagger"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/request-tagger.js",
```
Comment (marie-j) [inline packages/manager/modules/request-tagger/package.json]: Not an `index.js` ?

**Comment by cbourgois** [comment]
Force pushed : I've removed extra space in `BREAKING CHANGE` commit message

**Comment by cbourgois** [comment]
Rebased on `develop` to avoid conflicts.

**Pull request merged**